content/acra/configuring-maintaining/general-configuration/acra-server/encryptor-config.md

```diff
@@ -0,0 +1,456 @@
+
+```
+defaults:
+  crypto_envelope: "<acrablock|acrastruct>" # [optional] [default=acrablock] 
+  reencrypting_to_acrablocks: true # [optional] [default=true]
+```
+
+### **crypto_envelope**
+
+Required: `false`
+
+Type: `string` 
+
+Default value: `acrablock`
+
+Description: select which cryptographic algorithm and crypto envelope to use to encrypt fields. Don't change if you 
+don't know what it means. Supports: `acrablock` (default), `acrastruct`. 
+[Read more](/acra/acra-in-depth/data-structures/) about crypto envelopes, their purpose and difference.
+
+### **reencrypting_to_acrablocks**
+
+Required: `false`
+
+Type: `boolean`
+
+Default value: `false`
+
+
+Description: turns on the re-encryption of AcraStructs generated on application side to AcraBlocks before passing it to database. 
+It replaces less performant AcraStructs to the [more performant AcraBlocks](/acra/configuring-maintaining/optimizations/acrastructs_vs_acrablocks/).
+
+If application generates AcraStructs, writes once and reads often then we suggest to set `reencrypting_to_acrablocks: true`.
+It makes slower write operations due to decryption AcraStructs generated on application side and encrypting into 
+AcraBlocks on AcraServer side. But it significantly improves read operations performance due to faster decryption KEK 
+with symmetric cryptography in comparison with asymmetric.
+
+If application doesn't generate AcraStructs and rely only on transparent encryption on AcraServer side then we suggest
+turn it off. AcraServer will not try to recognize AcraStruct on every AcraBlock matching failure and slightly improve
+performance for write operations.
+
+[Read more](/acra/acra-in-depth/data-structures/) about crypto envelopes, their purpose and difference.
+
+## **schemas section**
+
+This section defines the table schema, and how AcraServer should process each column field. This field is array
+with items:
+```
+schemas:
+  - table: <string>
+    columns: <array of strings>
+    encrypted: <array of structured items>
+  - table: <string>
+    columns: <array of strings>
+    encrypted: <array of structured items>
+```
+
+### **table section**
+
+Required: `true`
+
+Type: `string`
+
+Description: a table name which columns will be processed (encrypted, tokenized, etc).
+{{< hint warning >}}
+Keep in mind that AcraServer 0.93.0 and earlier processes table name as case-sensitive identifier. Thus, "emails" and "Emails" are processed as different tables.
+{{</ hint  >}}
+
+### **columns section**
+
+Required: `false`
+
+Type: `array of strings`
+
+Description: defines all table's columns in order how they were declared in the database. AcraServer works as fully transparent proxy and doesn't access to the database's metadata with tables' schemas. 
```

Comment:
nit: I would wrap it into `warning` clause

Reply:
wrapped into warning last sentence to get attention on that